src/utils/components/PolicyForm/PolicyWizard/components/InterfaceDetailsExpandable.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC } from 'react';
+import { Updater } from 'use-immer';
+
+import {
+  Button,
+  FormFieldGroupExpandable,
+  FormFieldGroupHeader,
+  Tooltip,
+} from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+import {
+  InterfaceType,
+  NodeNetworkConfigurationInterface,
+  V1NodeNetworkConfigurationPolicy,
+} from '@types';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+
+import { INTERFACE_TYPE_LABEL, onInterfaceChangeType } from '../../constants';
+import { getExpandableTitle, isOVSBridgeExisting } from '../../utils';
+
+import InterfaceDetails from './InterfaceDetails';
+
+type InterfaceDetailsExpandableProps = {
+  policy: V1NodeNetworkConfigurationPolicy;
+  setPolicy: Updater<V1NodeNetworkConfigurationPolicy>;
+  interfaceTypes: InterfaceType[];
+};
+
+const InterfaceDetailsExpandable: FC<InterfaceDetailsExpandableProps> = ({
```

Comment:
Consider `InterfaceDetailsExpandableSection` or something similar. Ending in 'Expandable' just sounds weird for some reason. =)